jerry-debugger/jerry_client_main.py

```diff
@@ -208,20 +212,20 @@ def __init__(self, is_func, byte_code_cp, source, source_name, line, column, nam
 
         for i, _line in enumerate(lines):
             offset = offsets[i]
-            breakpoint = JerryBreakpoint(_line, offset, self)
-            self.lines[_line] = breakpoint
-            self.offsets[offset] = breakpoint
+            br_point = JerryBreakpoint(_line, offset, self)
+            self.lines[_line] = br_point
+            self.offsets[offset] = br_point
 
     def __repr__(self):
         result = ("Function(byte_code_cp:0x%x, source_name:%r, name:%r, line:%d, column:%d { "
                   % (self.byte_code_cp, self.source_name, self.name, self.line, self.column))
 
-        result += ','.join([str(breakpoint) for breakpoint in self.lines.values()])
+        result += ','.join([str(br_point) for br_point in list(self.lines.values())])
```

Comment:
Why do we need the `list` call here? the `.values()` should already be an iterable object.

Reply:
this question is still open.